infrastructure/environments/demo-cfn-update-args.yaml

```diff
@@ -1,9 +1,9 @@
-TemplateURL: https://pcluster-manager-github-infrastructurebucket-7y5h202iem8l.s3.eu-west-1.amazonaws.com/parallelcluster-ui.yaml
+TemplateURL: BUCKET_URL_PLACEHOLDER/parallelcluster-ui.yaml
 Parameters:
   - ParameterKey: Version
     ParameterValue: 3.4.0
   - ParameterKey: InfrastructureBucket
-    ParameterValue: https://pcluster-manager-github-infrastructurebucket-7y5h202iem8l.s3.eu-west-1.amazonaws.com
+    ParameterValue: BUCKET_URL_PLACEHOLDER
```

Comment:
suggestion: there's no need anymore to specify the `InfrastructureBucket` parameter. It can be switched to `UsePreviousValue: true` and the `TemplateURL` can be passed directly to the `cloudformation update-stack` command to avoid doing any `sed` operation